src/ResourceGraph/ResourceGraph/Az.ResourceGraph.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.7.1'
+ModuleVersion = '0.7.2'
```

Comment:
@demyanenko you can revert this change -- we'll version this module and publish it during our next release